modules/python/src2/cv2.cpp

```diff
@@ -933,27 +954,6 @@ PyObject* pyopencv_from(const Point3f& p)
 }
 
 template<>
-bool pyopencv_to(PyObject* obj, Vec3d& v, const char* name)
-{
-    (void)name;
-    if(!obj)
-        return true;
-    return PyArg_ParseTuple(obj, "ddd", &v[0], &v[1], &v[2]) > 0;
-}
-
-template<>
-PyObject* pyopencv_from(const Vec3d& v)
-{
-    return Py_BuildValue("(ddd)", v[0], v[1], v[2]);
```

Comment:
BTW: I have already tested various examples in the `samples/native/python` directory, and they are doing just fine!